pykechain/models/activity.py

```diff
@@ -828,7 +837,8 @@ def download_as_pdf(self, target_dir=None, pdf_filename=None, paper_size=PaperSi
         request_params = {
             'papersize': paper_size,
             'orientation': paper_orientation,
-            'appendices': include_appendices
+            'appendices': include_appendices,
```

Comment:
I saw you re-instantiated the cassettes, but can you also add this option to the tests?